test/components/hellip_button_test.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class PrimerHellipButtonTest < Minitest::Test
+  include Primer::ComponentTestHelpers
+
+  def test_renders
+    render_inline(Primer::HellipButton.new("aria-label": "No effect"))
+
+    assert_selector("button[type='button'][aria-expanded='false']", text: "…")
+  end
+
+  def test_renders_inline
+    render_inline(Primer::HellipButton.new(inline: true, "aria-label": "No effect"))
+
+    assert_selector("button[type='button'][aria-expanded='false']", text: "…")
```

Comment:
This assertion is the same as line 12. Shouldn't it be different?

Reply:
Good catch! I have changed it to include the `inline` class that's added.